packages/now-next/src/index.ts

```diff
@@ -336,13 +336,17 @@ export const build = async ({
   const routesManifest = await getRoutesManifest(entryPath, realNextVersion);
   const rewrites: Route[] = [];
   const redirects: Route[] = [];
+  let nextBasePath: string | undefined;
 
   if (routesManifest) {
     switch (routesManifest.version) {
       case 1:
       case 2: {
         redirects.push(...convertRedirects(routesManifest.redirects));
         rewrites.push(...convertRewrites(routesManifest.rewrites));
+        if (routesManifest.basePath) {
+          nextBasePath = routesManifest.basePath;
```

Comment:
We should add an `invariant` here that the `basePath` starts with a `/` and does not end with a trailing `/`. Also, the value `/` should be special cased and ignored as that's the default.

Reply:
Added some invariants and special cased the default `/`